api/processors/auxil/algorithms.py

```diff
@@ -0,0 +1,171 @@
+# DO NOT USE IN PROD, WORK IN PROGRESS
+
```

Comment:
Thanks, I'll take a look @Roxe322 . I definitely know now that creating `/aux/` modules is a very nasty anti-pattern as it makes code that is run under Windows break 😆

Reply:
I agree with using sorts of "utils" being an antipattern. At the same time I haven't seen any extensive codebases really being able to avoid it completely. So Ithink we should try avoiding this, but still cut ourselves some slack if it takes too much time.

Reply:
> I suggest to do api/processors/services/group_builder.py or even without /services/ and also rename GroupBuilderAlgorithm to GroupBuilderService, or just GroupBuilder.

That's exactly how it was in the first attempt (not in terms of path, but in terms of naming module and class). Before I rename, can @Roxe322 and @lemontree210 agree on this, please?

Reply:
@lemontree210 What do you think about 
`api/processors/services/group_builder.py:GroupBuilder`

Reply:
> @lemontree210 What do you think about `api/processors/services/group_builder.py:GroupBuilder`

OK, let's make it a `Builder`. I think it is rather easy to understand the difference between `Builder` and `Processor`.

Reply:
> util

You know what? I just read this article 😆 It actually says that an `utils` **module** is bad, not a package with descriptive names. If we have utils for several packages and put them into separate sub-packages of the respective packages, storing them in modules with descriptive names, then this is not really an anti-pattern, is it?

Reply:
> api/processors/services/group_builder.py:GroupBuilder

Done!